packages/shared/src/domain/customer.ts

```diff
@@ -1,16 +1,27 @@
 import { z } from "zod";
 
+export const orgDataSchema = z.object({
+  id: z.string(),
+  oid: z.string(),
+  cxId: z.string(),
+  name: z.string(),
+  type: z.string(),
+  businessType: z.string(),
```

Comment:
Let's make these match the strong type? Based on `OrgType` and `OrganizationBizType`.